src/soc/ibm/power9/istep_13_11.c

```diff
@@ -145,7 +145,7 @@ static void dump_cal_errors(uint8_t chip, int mcs_i, int mca_i)
 		printk(BIOS_ERR, "DP %d\n", dp);
 		printk(BIOS_ERR, "\t%#16.16llx - RD_VREF_CAL_ERROR\n",
 		       dp_mca_read(chip, id, dp, mca_i, DDRPHY_DP16_RD_VREF_CAL_ERROR_P0_0));
-		printk(BIOS_ERR, "\t%#16.16llx - DQ_BIT_DISABLE_RP0\n",
+		printk(RAM_DEBUG, "\t%#16.16llx - DQ_BIT_DISABLE_RP0\n",
```

Comment:
Please change all of lines printed by this function to `RAM_SPEW` and remove `#if CONFIG(DEBUG_RAM_SETUP)`

Reply:
Zrobiłem. Jeszcze ew. w ccs.c jest taki sam blok, też go tak zmodyfikować?